src/WARCSplitReader.py

```diff
@@ -4,8 +4,8 @@
 import lxml.html as lh
 from lxml import etree
 import bs4
-from config import TMP_FOLDER, WARC_ID, WARC_PER_DOC
 
+WARC_ID = "WARC-TREC-ID"
```

Comment:
is this correct? should be controlable via env or arg right?

Reply:
@abijithanikkuruthi what's with this?

Reply:
Prof Urbani said he's gonna use only WARC-TREC-ID in one of the QnA sessions and hence the hardcode!

Reply:
👍 Okay, but add this assumption the execution path pls